homeassistant/components/goodwe/select.py

```diff
@@ -15,17 +15,26 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+_OPERATION_MODES: dict[OperationMode, str] = {
+    OperationMode.GENERAL: "General mode",
+    OperationMode.OFF_GRID: "Off grid mode",
+    OperationMode.BACKUP: "Backup mode",
+    OperationMode.ECO: "Eco mode",
+    OperationMode.PEAK_SHAVING: "Peak shaving",
+    OperationMode.ECO_CHARGE: "Eco charge mode",
+    OperationMode.ECO_DISCHARGE: "Eco discharge mode",
+}
+
+
+def _get_operation_mode(mode: str) -> OperationMode:
```

Comment:
Make another dict instead that's the inversion of the dict above.

Reply:
Done, inverter dict created.